docs/gettinginvolved/infrastructure/packaging.rst

```diff
@@ -146,3 +149,35 @@ To check the image, run the container with the argument ```--version```:
 .. prompt:: bash $
 
    docker run --rm clusterhq/flocker-dataset-agent:master --version
+
+To run the container:
+
+.. prompt:: bash $
+
+    docker run \
+        --net host \
+        --privileged \
+        --volume /flocker:/flocker:shared \
+        --volume /etc/flocker:/etc/flocker \
+        --volume /dev:/dev \
+        --detach \
+        clusterhqci/flocker-dataset-agent:master
+
+
+flocker-control
+---------------
```

Comment:
Add a note about how the `flocker-control` Docker image is built using the same `docker build ...` command line as for `flocker-dataset` but substituting the `control/Dockerfile`.